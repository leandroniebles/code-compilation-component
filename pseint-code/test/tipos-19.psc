Algoritmo sin_titulo
	Escribir no 1
	Escribir no .1
	Escribir no -1
FinAlgoritmo
