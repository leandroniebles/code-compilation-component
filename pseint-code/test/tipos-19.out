=== Line 2: SynError 291
=== Line 3: SynError 291
=== Line 4: SynError 291
